package.json

```diff
@@ -59,18 +60,24 @@
   "devDependencies": {
     "@babel/cli": "7.8.4",
     "@babel/core": "7.9.0",
+    "@babel/eslint-parser": "7.12.1",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.12.1",
     "@babel/plugin-transform-modules-commonjs": "7.12.1",
     "@babel/preset-react": "7.9.4",
+    "@babel/preset-typescript": "7.12.7",
     "@rollup/plugin-commonjs": "16.0.0",
     "@rollup/plugin-node-resolve": "7.1.3",
+    "@types/jest": "26.0.20",
+    "@types/styled-system__css": "5.0.14",
```

Comment:
Are these styled-system types files truly dev dependencies? I see that `@types/styled-system` is included in dependencies, so just wondering if they should be co-located.

Reply:
Good call. `@types/styled-system__*` should be regular dependencies. I'll move them.